### PR TITLE
SetVar Position Fix

### PR DESCRIPTION
**Pull request by cmkuo12**
- Fixed the offset of setvar block and problems with the setvar text positioning
- "ToString" function doesn't work very well with customizing text position, so I positioned "setvar" in objectmesh
- Menu blocks are still instantiated initially

### CODE REVIEW

File: Assets/Scripts/CodeBlockMenuManager.cs [hunk 1]
```diff
@@ -23,5 +27,9 @@ public class CodeBlockMenuManager : Singleton<CodeBlockMenuManager> {
         public void TurnMenuOn() {
             gameObject.SetActive(true);
```
Comment (tgroechel) [inline Assets/Scripts/CodeBlockMenuManager.cs:28]: you can just write `SetActive(true);` no need for `gameObject`; similar concept to not having to write `this` in c++

File: Assets/Scripts/CodeBlockMenuManager.cs [hunk 2]
```diff
@@ -23,5 +27,9 @@ public class CodeBlockMenuManager : Singleton<CodeBlockMenuManager> {
...
         }
+
+        public void TurnMenuOff() {
+            gameObject.SetActive(false);
```
Comment (tgroechel) [inline Assets/Scripts/CodeBlockMenuManager.cs:32]: same here

File: Assets/Scripts/ExerciseManager.cs [hunk 3]
```diff
@@ -59,8 +63,7 @@ public class ExerciseManager : Singleton<ExerciseManager> {
                     KuriManager.instance.SayAndDoPositiveAffect(KuriTextManager.TYPEOFAFFECT.Congratulation);
                     LoggingManager.instance.UpdateLogColumn(exerciseSubmissionResultCol, "Correct");
                     lastExerciseCompleted = true;
-                }
-                else {
+                } else {
```
Comment (tgroechel) [inline Assets/Scripts/ExerciseManager.cs:66]: put else on new line

File: Assets/Scripts/SetValueObjectMesh.cs [hunk 4]
```diff
@@ -26,23 +28,20 @@ public class SetValueObjectMesh : CodeBlockObjectMesh {
...
+                RepositionText();
                 StartCoroutine(UpdateTextNextFrame());
-            }
-            else {
+                textMeshSetVar = top.GetChild(1).GetComponent<TextMeshPro>();
+                textMesh = middle.GetChild(0).GetComponent<TextMeshPro>();
+            } else {
                 textMesh.SetText("To");
+                textMeshSetVar.SetText("Set\nVar");
                 // Forces Text update
```
Comment (tgroechel) [inline Assets/Scripts/SetValueObjectMesh.cs:40]: I actually just made an extension method for this, you can now do `textMesh.ForceTextUpdate()` see `Extensions.cs` for the method

**Review by tgroechel — CHANGES_REQUESTED** [review]
Super minor changes but looks good otherwise

File: Assets/Scripts/CodeBlockMenuManager.cs (hunk 1 above)
Comment (cmkuo12) [reply Assets/Scripts/CodeBlockMenuManager.cs:28]: setactvie

There's an error when I don't use `gameobject`, so I just left it

**Review by tgroechel — APPROVED** [review]

**Pull request merged**